core/src/avm2/globals/flash/net.rs

```diff
@@ -29,15 +30,20 @@ pub fn navigate_to_url<'gc>(
         .ok_or("navigateToURL: not enough arguments")?
         .coerce_to_string(activation)?;
 
-    let url = request
-        .get_public_property("url", activation)?
-        .coerce_to_string(activation)?;
-
-    activation.context.navigator.navigate_to_url(
-        &url.to_utf8_lossy(),
-        &target.to_utf8_lossy(),
-        None,
-    );
-
-    Ok(Value::Undefined)
+    match request.get_public_property("url", activation)? {
```

Comment:
Have you checked that this errors when calling `navigateToURL` and not when setting the property on `URLRequest`?

Reply:
Yes.